examples/http-request-errors.jsonnet

```
@@ -0,0 +1,20 @@
+local slo = import '../jsonnet/sla-mixin/mixin.libsonnet';
+
+{
+  local errors = slo.errors({
+    metric: 'promhttp_metric_handler_requests_total',
+    warning: 5,  // 5% of total requests
+    critical: 10,  // 10% of total requests
+    jobSelector: 'job: fooapp',
+    namespaceSelector: 'namespace: default',
```

Let's call this simply `selectors`:
```suggestion
    selectors: 'namespace="default",job="fooapp"',
```

We need this to be independent of anything we are aware of, as some other apps might have totally arbitrary selectors. Example, the kubernetes-mixin needs `{job="kube-apiserver",quantile="0.99",subresource!="log",verb!~"^(?:LIST|WATCH|WATCHLIST|PROXY|CONNECT)$"}` as selector. :blush: